dist/pom.xml

```diff
@@ -55,64 +55,56 @@
             </exclusions>
```

Comment:
Just a minor note. I noticed the `dist` module is `<packaging>jar</packaging>`. If we change it to `pom` it would stop the JAR from being created.

Reply:
good point.